pixeltable/catalog/table_version.py

```diff
@@ -634,7 +638,7 @@ def add_columns(
         all_cols: list[Column] = []
         for col in cols:
             all_cols.append(col)
-            if self._is_btree_indexable(col):
+            if col.name is not None and self._is_btree_indexable(col):
```

Comment:
Should we alias `col.name is not None` with something like `col.is_anonymous`?

Reply:
We should really add a separate flag. The names are useful for debugging.